Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -466,6 +466,7 @@ - (void)flushWithMaxSize:(NSUInteger)maxBatchSize
     [self dispatchBackground:^{
         if ([self.queue count] == 0) {
             SEGLog(@"%@ No queued API calls to flush.", self);
+            [self endBackgroundTask];
```

Comment:
@maxmeyers what's the rationale for this line? Why aren't the other two `endBackgroundTask` calls sufficient?